requirements.txt

```diff
@@ -1,4 +1,4 @@
-prefect>=2.7.2
+prefect>=2.13.5
 dbt_core>=1.1.1
-prefect_shell>=0.1.0
+prefect>=2.13.5
```

Comment:
@urimandujano Something got a little weird here, perhaps because there were two dependencies starting with `prefect`?

Reply:
AH yup -- I'll fix this one manually. Good eye!